### PR TITLE
Changed: Update toggle component and filter attribute

### LINKED ISSUE #6483: Toggle component should not compare translated labels against non-translated strings
## Details

Within the Toggle component used in the Donor Wall, we are comparing a translated label against a non translated string. There are also some concerns with naming conventions in the block attributes. Specifically for the filter attribute as "filter" is also a property within the shortcodes.

## Acceptance Criteria

-   [x] filter attribute name is changed.
-   [x] translated strings are not compared to non-translated strings.
-   [x] Toggle component still alternates between element displays.
-   [x] FormTokenField options still work properly.

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
Resolves #6483

## Description

This PR updates the filter attribute name and passes a new set of options to be  used within the Toggle component for the Donor Wall. 

## Affects

Donor Wall
Toggle block component

## Visuals

https://user-images.githubusercontent.com/75056371/175352455-24c131ce-8493-4d7d-a3e3-45c02e3a29d2.mov

## Testing Instructions

- Use the Donor Wall block or shortcode
- Select the block and open the settings for "Display Elements"
- Toggle between both options to verify that the selected options background color changes & that the list of toggles change
- Open the "Wall Settings" and use the filterable options to verify they continue to work as expected.
- 
## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit and/or end-to-end tests
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: blocks/donor-wall/class-give-donor-wall.php [hunk 1]
```diff
@@ -226,7 +226,7 @@ public function render_block( $attributes ) {
 			'readmore_text'     => $attributes['readMoreText'],
 			'loadmore_text'     => $attributes['loadMoreText'],
             'toggle_options'    => $attributes['toggleOptions'],
-            'filter'            => $attributes['filter'],
+            'filter_options'    => $attributes['filterOptions'],
```
Comment (ravinderk) [inline blocks/donor-wall/class-give-donor-wall.php:229]: @JoshuaHungDinh you do not need to pass  `filter_options ` and  `toggle_options ` in attributes to render donor wall shortcode. Can we remove them from here?

You can keep them in the block attributes list.

**Review by ravinderk — CHANGES_REQUESTED** [review]
@JoshuaHungDinh Pull request is good. I requested a minor change. Let me know if you have any questions.

File: blocks/donor-wall/class-give-donor-wall.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply blocks/donor-wall/class-give-donor-wall.php]: @ravinderk  Got it, I see what you mean. I made the same updates to the Form Grid as well.

**Review by ravinderk — APPROVED** [review]

**Comment by ravinderk** [comment]
@JoshuaHungDinh can you resolve conflict in this pull request?

**Pull request merged**